packages/components/src/lib/draw/index.ts

```diff
@@ -4,7 +4,7 @@ import { Padding, Point } from "./types.js";
 
 const axisColor = "rgba(114, 125, 147, 0.1)";
 export const labelColor = "rgb(114, 125, 147)";
-export const cursorLineColor = "#f87171"; // tailwind red-400
+export const cursorLineColor = "#aaa";
```

Comment:
Maybe it's just me, but this share of gray on blue causes me some eyestrain:
<img width="136" alt="Captura de pantalla 2023-05-28 a la(s) 18 21 06" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/1c1c4565-1670-41c0-8181-c1ab1a75cb98">
I think it's too low-contrast. I agree that red wasn't the best choice either, though.

Reply:
I can change to a darker gray. I expect it to be tricky to get right - too light and you can't see it against white, too dark and you can't see it well against the dark blue dists.

Reply:
It would also be interesting to experiment with [blend modes](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation) for this.